script/client

```diff
@@ -138,11 +147,32 @@ else {
 }
 
 $url->path($path);
-$url->query([%params]) if %params;
+
+if (!$options{'form'}) {
```

Comment:
just because perlcritic doesn't check scripts doesn't mean you can use quotes in hashes :)